tools/makebadges.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Generates markdown for README build status badges.
+#
+# Copyright 2025 James E. King III <jking@apache.org>
+#
+
+set -eu
+
+ARGS=$(getopt --longoptions "project:,appveyor:,codecov:,coverity:" "p:a:v:y:" -- "$@")
+
+if [ $? != 0 ] ; then echo "Terminating..." >&2 ; exit 1 ; fi
+
+eval set -- "$ARGS"
+
+PROJECT=
+APPVEYOR=
+CODECOV=
+COVERITY=
+
+while true; do
+  case "${1:-*}" in
```

Comment:
The default of `*` looks odd, I'd use an empty value. But as any value works this is fine.